pkg/util/util_test.go

```diff
@@ -11,10 +11,11 @@ import (
 	"github.com/google/gopacket/examples/util"
 	"github.com/google/gopacket/layers"
 	log "github.com/sirupsen/logrus"
+	"github.com/wencaiwulue/kubevpn/pkg/errors"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-func TestName(t *testing.T) {
+func ConfigCNI(t *testing.T) {
```

Comment:
is this ok to change?

Reply:
Keep Test Prefix this golang  requirement, otherwise, this function will not run when you use `go test -v -failfast ./...`